artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py

```diff
@@ -636,10 +642,35 @@
     "CVE-2024-24919 is an information disclosure vulnerability that can allow an attacker to access certain information on internet-connected Gateways which have been configured with IPSec VPN, remote access VPN or mobile access software blade.": "Wykryto podatność CVE-2024-24919 w urządzeniu Check Point Quantum Gateway umożliwiającą atakującemu odczyt dowolnych plików z dysku.",
     "ELMAH (Error Logging Modules and Handlers) is an application-wide error logging facility that is completely pluggable. It can be dynamically added to a running ASP.NET web application, or even all ASP.NET web applications on a machine, without any need for re-compilation or re-deployment. In some cases, the logs expose ASPXAUTH cookies allowing to hijack a logged in administrator session.": "Wykryto, że dziennik zdarzeń udostępniany przez moduł ELMAH (Error Logging Modules and Handlers) jest publicznie dostępny. W niektórych sytuacjach może on zawierać informacje umożliwiające przejęcie zalogowanej sesji administratora.",
     "Checks websites for Balada Injector malware.": "Wykryto, że strona jest zainfekowana złośliwym oprogramowaniem Balada Injector.",
+    "WS_FTP software, which is a popular FTP (File Transfer Protocol) client used for transferring files between a local computer and a remote server has its log file exposed.": "Wykryto dziennik zdarzeń oprogramowania WS_FTP.",
     "[no description] http/exposed-panels/compalex-panel-detect.yaml": "Wykryto panel Compalex.",
     "Joomla is susceptible to the Installation page exposure due to misconfiguration.": "Wykryto publicznie dostępny panel instalacyjny systemu Joomla.",
     "Exposed Wordpress Setup Configuration.": "Wykryto publicznie dostępny panel instalacyjny systemu WordPress.",
     "Error log files were exposed.": "Wykryto publicznie dostępny dziennik zdarzeń serwera HTTP." + DATA_HIDE_HINT,
+    "Multiple compressed backup files were detected.": "Wykryto publicznie dostępny plik kopii zapasowej."
+    + DATA_HIDE_HINT,
+    "Multiple Docker Compose configuration files were detected. The configuration allows deploy, combine and configure operations on multiple containers at the same time. The default is to outsource each process to its own container, which is then publicly accessible.": "Wykryto plik docker-compose.yml, umożliwiający atakującemu poznanie konfiguracji aplikacji."
```

Comment:
```suggestion
    "Multiple Docker Compose configuration files were detected. The configuration allows deploy, combine and configure operations on multiple containers at the same time. The default is to outsource each process to its own container, which is then publicly accessible.": "Wykryto pliki konfiguracyjne Docker Compose, umożliwiający atakującemu poznanie konfiguracji aplikacji."
```

because it might be found in other endpoint than `docker-compose.yml`